Core/Config.h

```diff
@@ -153,6 +153,9 @@ struct Config {
 
 	// GFX
 	int iGPUBackend;
+	// We have separate device parameters for each backend so it doesn't get erased if you switch backends.
+	// If not set, will use the "best" device.
+	std::string VulkanDevice;
```

Comment:
sVulkanDevice and sD3D11Device?  We're kinda stuck with the convention now.

-[Unknown]

Reply:
Yea, good point.